carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/utils/android/AndroidService.java

```diff
@@ -1207,10 +1208,11 @@ private void setDeviceTimeZoneBySetting(String timeZone, String settingsTZ, Time
             openDateTimeSettingsSetupWizard(true, timeFormat);
 
             String res = getCurrentDeviceFocus();
-            if (res.contains("settings.DateTimeSettingsSetupWizard")) {
-                LOGGER.info("On settings.DateTimeSettingsSetupWizard page");
+
+            if (res.contains("settings.Settings$DateTimeSettingsActivity")) {
```

Comment:
Probably should be separated for Android 8 and older version.

Reply:
It's backwards compatible, so should work on all versions (verified on 6.x/7.x/8.x).